collect_app/src/androidTest/assets/forms/one-question-entity.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:entities="http://www.opendatakit.org/xforms/entities">
+	<h:head>
+		<h:title>One Question Entity</h:title>
+		<model odk:xforms-version="1.0.0" entities:entities-version="2022.1.1">
+			<instance>
+				<data id="one_question_entity" version="1">
+					<name/>
+					<meta>
+						<instanceID/>
+						<instanceName/>
+						<entities:entity entities:dataset="people">
```

Comment:
After reading https://www.w3.org/TR/2006/REC-xml-names11-20060816/#defaulting: "Default namespace declarations do not apply directly to attribute names; the interpretation of unprefixed attributes is determined by the element on which they appear." and https://stackoverflow.com/a/46865/137744, we've decided not to namespace the attributes on the `entity` block. I think I checked for that in the JR pull request and made sure they're not enforced but would be worth a double check and to change this here.

`saveto` and `entities-version` get namespaced because they are on nodes that are not "ours".

Reply:
I think I see what you're getting at, but just to confirm, you mean that `dataset` shouldn't need a prefix? I'm guessing the inner nodes (`create` and `label`) still do however?